src/frontendHighlighterApp/sass/app.scss

```
@@ -159,34 +202,144 @@ body {
+			}
+
+			.edac-badge__icon {
+				display: inline-flex !important;
+				align-items: center !important;
+
+				svg {
+					display: block !important;
+					width: 16px !important;
+					height: 16px !important;
+					overflow: visible !important;
+					stroke: currentColor !important;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)